addon/globalPlugins/remoteClient/local_machine.py

```diff
@@ -77,4 +84,14 @@ def set_clipboard_text(self, text, **kwargs):
 		api.copyToClip(text=text)
 
 	def send_SAS(self, **kwargs):
-		ctypes.windll.sas.SendSAS(0)
+		"""
+		This function simulates as "a secure attention sequence" such as CTRL+ALT+DEL.
+		SendSAS requires UI Access, so we provide a warning when this fails.
+		This warning will only be read by the remote NVDA if it is currently connected to the machine.
+		"""
+		if systemUtils.hasUiAccess():

```

Comment:
After latest changes this should be:
```suggestion
		if hasUiAccess():
```